app/templates/editions/octane.hbs

```diff
@@ -100,11 +70,15 @@
   <dl class="dl-faq">
     <dt>Which release of Ember is Octane?</dt>
     <dd>
-      The Octane Edition was declared for Ember 3.15.
+      The Octane Edition was declared for [Ember 3.15 in December 2019](https://blog.emberjs.com/ember-3-15-released).
```

Comment:
`declared for` (currently present) sounded a bit awkward to me. Could we switch the order of the words?

```suggestion
      In December 2019, [Ember 3.15](https://blog.emberjs.com/ember-3-15-released) was declared to be Ember Octane.
```